### ISSUE TITLE
[v3] TextInput's numberOfLines not working 

### ISSUE THREAD

**Issue by vkasraj** [callstack/react-native-paper#1400]
## Environment
"expo": "^35.0.0",
"formik": "^1.5.8",
react-native: 0.59.8
"react-native-paper": "^3.0.0-alpha.7"

## Description
`numberOfLines` has not effect on `<TextInput />`. But `multiline` was working fine.

#### Form Component
```ts
import React from "react";
import { Formik, Field } from "formik";
import { View, StyleSheet } from "react-native";
import FormikTextField from "./Fields/FormikTextField";
import { Button } from "react-native-paper";

const initialValues = {
    title: "",
    description: "",
    pinned: false,
    archived: false
};

const NoteForm = ({ onSubmit }) => {
    return (
        <Formik onSubmit={onSubmit} initialValues={initialValues}>
            {({ handleSubmit }) => {
                return (
                    <View>
                        <Field
                            component={FormikTextField}
                            name="title"
                            label="Title"
                        />
                        <Field
                            component={FormikTextField}
                            name="description"
                            label="Description"
                            multiline={true}
                            numberOfLines={10}
                        />
                        <Button
                            mode="contained"
                            onPress={handleSubmit}
                            compact
                            style={styles.mt}
                        >
                            Create Note
                        </Button>
                    </View>
                );
            }}
        </Formik>
    );
};

const styles = StyleSheet.create({
    mt: {
        marginTop: 20
    }
});

export default NoteForm;
```
### FormikTextField
```ts
import React, { FC } from "react";
import { FieldProps } from "formik";
import { TextInput } from "react-native-paper";
import { StyleSheet } from "react-native";

const FormikTextField: FC<FieldProps> = ({ field, form, ...props }) => {
    const { name } = field;
    const { setFieldValue, handleBlur, values } = form;

    return (
        <TextInput
            style={styles.mb}
            onChangeText={value => setFieldValue(name, value)}
            onBlur={() => handleBlur(name)}
            value={values[name]}
            {...field}
            {...field}
            {...props}
        />
    );
};

const styles = StyleSheet.create({
    mb: {
        marginBottom: 10
    }
});

export default FormikTextField;
```

## ScreenShot
![Screenshot_20191021-121832](https://user-images.githubusercontent.com/24727447/67183108-4c5e4500-f3fe-11e9-9a71-c778b2ebae7f.jpg)

## Device
**Redmi Note 7 Pro**
![Screenshot_20191021-122954](https://user-images.githubusercontent.com/24727447/67183217-93e4d100-f3fe-11e9-9ddc-3ba7c94170e3.jpg)

**Comment by venits** [comment]
I can confirm that there is something wrong.

When I test it locally by forking this repo and playing with **example** folder it works.
When I set up new expo project it won't work ;/

There might be a problem with building and packaging this library.

For now solution will be to set height for you component like this:

```js
 <Field
   ...
  style={{ height: 100 }}
/>
```

**Comment by venits** [comment]
I created pull request for it:
https://github.com/callstack/react-native-paper/pull/1401

**Comment by Trancever** [comment]
Should be fixed in 3.0.0-alpha.8
